frontend/components/molecules/MForgotPasswordDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC } from 'react';
+import { Box, Dialog, IconButton, TextField, Grid, DialogContent, Button, FormControl } from '@material-ui/core';
+import { useState, ChangeEvent } from 'react';
+import { Close } from '@material-ui/icons';
+
+interface MForgotPasswordDialogProps {
+  showForgotPasswordModal?: boolean;
+  handleCloseForgotPasswordModal?: Function;
+}
+
+const MForgotPasswordDialog: FC<MForgotPasswordDialogProps> = ({
+  showForgotPasswordModal = false,
+  handleCloseForgotPasswordModal = () => {},
+}) => {
+  const [email, setEmail] = useState<String>('');
+
+  const handleEmailChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  const handleSubmit = () => {
+    alert(`You have entered ${email}`);
+    setEmail('');
+  };
+
+  return (
+    <Dialog open={showForgotPasswordModal} fullWidth>
+      <Box display="flex" justifyContent="flex-end" mt={2} mr={2}>
+        <IconButton aria-label="close" onClick={() => handleCloseForgotPasswordModal()} color="inherit">
+          <Close />
+        </IconButton>
+      </Box>
+      <DialogContent>
+        <Grid container direction="column" spacing={6}>
+          <Grid item>
+            <Box component="h1">Forgot Password</Box>
+          </Grid>
+          <Grid item>
+            <FormControl fullWidth>
```

Comment:
We do not need FormControl as we have our own MPasswordField / TextField components we can use.